ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2576,32 +2576,38 @@
     "description": "Default stack with Python 3.7 and Theia IDE",
     "components": [
       {
-        "version": "9",
-        "name": "Debian"
+        "version": "7.6",
+        "name": "CentOS"
       },
       {
-        "version": "3.7.3",
+        "version": "3.6.3",
         "name": "Python"
       },
       {
-        "version": "19.0.3 ",
+        "version": "9.0.1",
         "name": "PIP"
       }
     ],
     "workspaceConfig": {
       "environments": {
         "default": {
           "recipe": {
-            "content": "python:3.7.3",
+            "content": "centos/python-36-centos7:1",
             "type": "dockerimage"
           },
           "machines": {
             "python": {
+              "servers": {
+                "django": {
```

Comment:
Do we really need a server? Isn't the [port plugin](https://github.com/eclipse/che-theia/tree/master/plugins/ports-plugin) exposing it automatically?

Reply:
Yes, it is. But after a little application becomes unavailable.

Reply:
Are you saying that the application is unstable when using the port plugin but works just fine if you define a server?

Reply:
Exactly

Logs from theia-ide container (minishift):
```
root ERROR Uncaught Exception:  Error [ERR_IPC_CHANNEL_CLOSED]: Channel closed
--
  | root ERROR Error [ERR_IPC_CHANNEL_CLOSED]: Channel closed
  | at ChildProcess.target.send (internal/child_process.js:636:16)
  | at HostedPluginProcess.onMessage (/home/theia/node_modules/@theia/plugin-ext/lib/hosted/node/hosted-plugin-process.js:105:31)
  | at ServerPluginProxyRunner.onMessage (/home/theia/node_modules/@eclipse-che/theia-remote/lib/node/server-plugin-proxy-runner.js:49:32)
  | at /home/theia/node_modules/@theia/plugin-ext/lib/hosted/node/hosted-plugin.js:99:28
  | at Array.forEach (<anonymous>)
  | at HostedPluginSupport.onMessage (/home/theia/node_modules/@theia/plugin-ext/lib/hosted/node/hosted-plugin.js:97:32)
  | at HostedPluginServerImpl.onMessage (/home/theia/node_modules/@theia/plugin-ext/lib/hosted/node/plugin-service.js:152:27)
  | at JsonRpcProxyFactory.onNotification (/home/theia/node_modules/@theia/core/lib/common/messaging/proxy-factory.js:246:36)
  | at /home/theia/node_modules/@theia/core/lib/common/messaging/proxy-factory.js:181:53
  | at handleNotification (/home/theia/node_modules/vscode-jsonrpc/lib/main.js:483:

```

Reply:
https://github.com/eclipse/che/issues/13170